.ripgreprc

```diff
@@ -0,0 +1,2 @@
+--hidden
+--glob=!.git/*
```

Comment:
Echoing David's comment, I wasn't sure about this file, but found this:
https://github.com/BurntSushi/ripgrep
Interesting tool, I haven't heard of this one before.